tests/Tests/200__monitor_and_manage/210__alerts/210__alerts.robot

```diff
@@ -469,3 +484,34 @@ Delete Failed Build And Start New One
     ${build_name} =    Start New Build    namespace=${namespace}
     ...    buildconfig=${build_config_name}
     Wait Until Build Status Is    namespace=${namespace}    build_name=${build_name}
+
+Check Cluster Name Contain "Aisrhods" Or Not
+    [Documentation]     Return true if cluster name contains aisrhods and it not return false
+    ${cluster_id} =     Get Cluster ID
+    ${cluster_name} =    Get Cluster Name By Cluster ID  cluster_id=${cluster_id}
+    IF    "aisrhods" in "${cluster_name}"
+        ${return_value} =    Set Variable    ${TRUE}
+    ELSE
+        ${return_value} =    Set Variable    ${FAlSE}
+    END
+    [Return]    ${return_value}
+
+Check Particular Text Is Present In Rhods-operator's Log
+    [Documentation]     Check Is text is present in log
+    [Arguments]         ${text_to_check}
```

Comment:
@tarukumar didn't you already have a keyword to do something similar to this?

Reply:
@lugi0 Yes, I have a TC which looks for the error msg in operator log(NO keyword defined). However, I think this can be also useful since he is using UI to capture the logs.

Reply:
Thanks @tarukumar and @lugi0